src/pylexibank/dataset_templates/lexibank_simple/lexibank_+id+.py_tmpl

```diff
@@ -0,0 +1,81 @@
+from clldutils.path import Path
+import pylexibank
+
+# Customize your basic data.
+# if you need to store other data in columns than the lexibank defaults, then over-ride
+# the table type (pylexibank.[Language|Lexeme|Concept|Cognate|]) and add the required columns e.g.
+#
+#import attr
+#
+#@attr.s
+#class Concept(pylexibank.Concept):
+#    MyAttribute1 = attr.ib(default=None)
+
+
+class Dataset(pylexibank.Dataset):
+    dir = Path(__file__).parent
+    id = "{id}"
+
+    # register custom data types here (or language_class, lexeme_class, cognate_class):
+    #concept_class = Concept
+
+    # define the way in which forms should be handled
+    form_spec = pylexibank.FormSpec(
+        brackets={{"(": ")"}},  # characters that function as brackets
```

Comment:
Due to template files being rendered by simply calling `"".format()`, any `{` or `}` in the content must be escaped as `{{` and `}}`. This is slightly awkward, but means trivial template rendering code.